pytorch_lightning/utilities/cli.py

```diff
@@ -772,12 +769,16 @@ def _prepare_subcommand_kwargs(self, subcommand: str) -> Dict[str, Any]:
         return fn_kwargs
 
 
-def _global_add_class_path(class_type: Type, init_args: Dict[str, Any] = None) -> Dict[str, Any]:
+def _global_add_class_path(
+    class_type: Type, init_args: Optional[Union[Namespace, Dict[str, Any]]] = None
```

Comment:
```suggestion
    class_type: Type, init_args: Optional[Namespace] = None
```

Reply:
Because of `_convert_argv_issue_84` and `_convert_argv_issue_85` this does receive a dict. This could changed once there are native features.